frontend/yarn.lock

```diff
@@ -1005,6 +1005,13 @@
   dependencies:
```

Comment:
hmm, just curious, do you think the `yarn.lock` file needs to be pushed onto git? or should it be included in the `.gitignore` file?
i think `yarn.lock` is automatically generated and it is not needed for dependency installation since that is based on `package.json`, but im not 100% sure.

Reply:
I think it does need to be included? I've been basically using yarn to download things instead of npm lol and a quick Google search says yes include it

Reply:
sounds good! thanks for looking it up :)